test/assertionStorage.test.ts

```diff
@@ -48,4 +59,86 @@ describe('AssertionStorage contract', function () {
     expect(getAssertionResponse.triplesNumber).to.be.equal(triplesNumber);
     expect(getAssertionResponse.chunksNumber).to.be.equal(chunksNumber);
   });
+
+  it('Set non owner to be new contract owner and create an assertion, expect to pass', async () => {
+    await Hub.setContractAddress('HubOwner', accounts[1].address);
+    const AssertionStorageWithNonOwnerAsSigner = AssertionStorage.connect(accounts[1]);
+
+    await AssertionStorageWithNonOwnerAsSigner.createAssertion(assertionId, size, triplesNumber, chunksNumber);
+    const getAssertionResponse = await AssertionStorage.getAssertion(assertionId);
+
+    expect(getAssertionResponse.size).to.be.equal(size);
+    expect(getAssertionResponse.triplesNumber).to.be.equal(triplesNumber);
+    expect(getAssertionResponse.chunksNumber).to.be.equal(chunksNumber);
+  });
+
+  it('Get assertion for non-existing assertionId, expect to get 0', async () => {
+    const getAssertionResponse = await AssertionStorage.getAssertion(nonExistingAssertionId);
+
+    getAssertionResponse.forEach((e) => {
+      expect(e.toString()).to.be.eql('0');
+    });
+  });
+
+  it('Get assertion timestamp, size, triples/chunks number for non-existing assertionId, expect to get 0', async () => {
+    const getTimestampResult = await AssertionStorage.getAssertionTimestamp(nonExistingAssertionId);
+    const getSizeResult = await AssertionStorage.getAssertionSize(nonExistingAssertionId);
+    const getTriplesNumber = await AssertionStorage.getAssertionTriplesNumber(nonExistingAssertionId);
+    const getChunksNumber = await AssertionStorage.getAssertionChunksNumber(nonExistingAssertionId);
+
+    expect(getTimestampResult.toString()).to.be.eql('0');
```

Comment:
Why are you using eql instead of equal?

Reply:
The difference between eql and equal is that:
- eql: deeply equal to the given value
- equal: strictly (===) equal to the given value
Because of that, I thought eql is better, but the logic is the same.